guides/routing.md

```diff
@@ -492,7 +492,7 @@ Phoenix defines two pipelines by default, `:browser` and `:api`, which can be us
 
 ### The `:browser` and `:api` Pipelines
 
-As their names suggest, the `:browser` pipeline prepares for routes which render requests for a browser. The `:api` pipeline prepares for routes which produce data for an api.
+As their names suggest, the `:browser` pipeline prepares for routes which render requests for a browser and the `:api` pipeline prepares for routes which produce data for an api.
```

Comment:
```suggestion
As their names suggest, the `:browser` pipeline prepares for routes which render requests for a browser, and the `:api` pipeline prepares for routes which produce data for an api.
```

Reply:
Yeah the comma definitely should be in there, thanks!